mapboxgl/viz.py

```diff
@@ -39,18 +39,13 @@ def __init__(self,
         """
         if access_token is None:
             access_token = os.environ.get('MAPBOX_ACCESS_TOKEN', '')
-        if style_url.startswith('mapbox:') and not access_token.startswith('pk'):
```

Comment:
I'd like to retain this token error to give feedback to users who want to create a map from a Mapbox Style URL but don't specify a valid access token.  

Could we add another check here to see if the type of object passed as a style is a `dict` or a `string`?  If it's a string, we can check if the style is a `mapbox://` format url, and if so, check for a token error.